components/DisplayFoodBagsList.jsx

```diff
@@ -6,16 +6,16 @@ import DisplayFoodBag from './DisplayFoodBag'
 import FoodBagService from '../modules/FoodBagService'
 
 const DisplayFoodBagsList = ({ navigation }) => {
-  const { foodbags, credentials } = useSelector(state => state)
+  const { foodbags } = useSelector((state) => state)
   useEffect(() => {
-    FoodBagService.index(credentials)
+    FoodBagService.index()
```

Comment:
Any particular reason for removing the credentials on this?

Reply:
We are storing credentials in headers and setting them with token. Credentials actually didn't do anything

Reply:
Alright. cool